### PR TITLE
[#164629862] Update ruby 2.6.2, postgres 11

### PR DESCRIPTION
**Pull request by jackpipe**

### CODE REVIEW

**Review by mlongerich — APPROVED** [review]
LGTM. Just some questions as I don't know the context. I noticed we are now disabling 'ipv6'. Why the change? Also, I see the note about Firefox has been there for awhile. Any plans to upgrade it? I think they're on Firefox 66 now.

File: circleci-ruby/2.6.2-stretch-pgclient11.1-node10.15.1-browsers/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+
+#### PG CLIENT
+
+ENV PG_VERSION 11
+
+RUN mkdir ~/.gnupg
+RUN chmod 600 ~/.gnupg
+RUN echo "disable-ipv6" >> ~/.gnupg/dirmngr.conf
+
```
Comment (jackpipe) [inline circleci-ruby/2.6.2-stretch-pgclient11.1-node10.15.1-browsers/Dockerfile:13]: Disable ipv6 support for gnupg, workaround as described here:
https://github.com/inversepath/usbarmory-debian-base_image/issues/9

**Comment by jackpipe** [comment]
ipv6 is a workaround to gnupg trying to bind to ipv6, even though docker may not be configured for it, and therefore failing.  It only affects gnupg
Yes we should look at updating firefox etc in the future.

File: circleci-ruby/2.6.2-stretch-pgclient11.1-node10.15.1-browsers/Dockerfile (hunk 1 above)
Comment (mlongerich) [reply circleci-ruby/2.6.2-stretch-pgclient11.1-node10.15.1-browsers/Dockerfile:13]: Thanks for the link and context.

**Pull request merged**